scavenger-api/build.gradle.kts

```diff
@@ -50,17 +50,17 @@ dependencies {
 configure<DependencyManagementExtension> {
     imports {
         mavenBom(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES)
-        mavenBom("org.springframework.cloud:spring-cloud-dependencies:2020.0.4")
     }
 }
 
 kotlin {
-    jvmToolchain(11)
+    jvmToolchain(17)
 }
 
 tasks.withType<KotlinCompile> {
     kotlinOptions {
         freeCompilerArgs = listOf("-Xjsr305=strict")
+        javaParameters = true
```

Comment:
finally why was this option added?

Reply:
for `spring-data-jdbc-plus`.
I refered this guide https://github.com/naver/spring-jdbc-plus/blob/29ef973d6fd1b66338683259dfd6423954110f00/guide-projects/plus-sql-java-kotlin-guide/build.gradle#L19